docs/tls.md

```diff
@@ -45,7 +45,7 @@ To disable HTTPS, configure your environment to use HTTP by setting the followin
 ```bash
 SERVER_NAME=http://localhost \
 MERCURE_PUBLIC_URL=http://localhost/.well-known/mercure \
-docker compose up --pull always -d --wait
+docker compose up --wait
```

Comment:
Pulling the image was not required here. At the time of these steps, the images should be both built/pulled. I can assume that this was done in order to maintain a uniform format for the `docker compose up` command.